alacritty_terminal/src/ansi.rs

```diff
@@ -868,18 +875,22 @@ where
                 unhandled(params);
             },
 
+            // TODO: 50 still supported? https://iterm2.com/documentation-escape-codes.html
```

Comment:
We should probably keep support for this I'd assume, even though 1337 should be used instead? I'm not sure which application uses this escape.

Reply:
1337 sounds like iterm